client/java/src/main/java/io/openlineage/client/metrics/StatsDMetricsBuilder.java

```diff
@@ -0,0 +1,32 @@
+/*
+/* Copyright 2018-2024 contributors to the OpenLineage project
+/* SPDX-License-Identifier: Apache-2.0
+*/
+
+package io.openlineage.client.metrics;
+
+import io.micrometer.statsd.StatsdMeterRegistry;
+import java.util.Map;
+
+/**
+ * StatsDMetricsBuilder is a class that implements the {@link MetricsBuilder} interface specifically
+ * for {@link StatsdMeterRegistry}. It provides methods to check if the required {@link
+ * StatsdMeterRegistry} class is available and build a {@link StatsdMeterRegistry} instance based on
+ * a configuration map.
+ *
+ * @see <a
+ *     href="https://docs.micrometer.io/micrometer/reference/implementations/statsD.html">StatsdConfig
+ *     and StatsD micrometer interface docs</a>
+ */
+public class StatsDMetricsBuilder implements MetricsBuilder<StatsdMeterRegistry> {
```

Comment:
`StatsDMeterRegistryBuilder` ?

Reply:
Taken this offline with Paweł and renamed the classes accordingly.